tests/tests/fee_tests.cpp

```diff
@@ -258,6 +258,21 @@ BOOST_AUTO_TEST_CASE(asset_claim_pool_test)
 
         };
 
+        const asset_object& core_asset = asset_id_type()(db);
+
+        if( db.head_block_time() <= HARDFORK_CORE_188_TIME )
+        {
+           // can't claim pool before hardfork
+           GRAPHENE_REQUIRE_THROW( claim_pool( alice_id, bobcoin_id, _core(200), core_asset), fc::exception );
```

Comment:
Please modify the unit test case to test exactly what we want to test.
The asset was created before enabling fee, so the pool should be empty here, so an exception should be thrown out even if it's after hard fork time.
Also another exception would be thrown out since Alice can not claim bob's asset's fee pool.

Reply:
There is a test for claiming a pool that doesn't belong to you in line [285](https://github.com/bitshares/bitshares-core/blob/ea912be5502c3b3114bda4373eb26bc2da37f63d/tests/tests/fee_tests.cpp#L285)

I've fixed the test that raises an exception before the hard fork to claim.
Now 
* trying to claim **before the hardfork** - which should fail due to hard fork time
* trying to claim **after the hardfork**, but **before funding** which should fail due to now funds
* trying to claim **after the hardfork** and **after funding**, but **from non-issuer account** which fails due to wrong account

Is that about right?